lab-david/__test__/photoalbum-router.test.js

```diff
@@ -0,0 +1,202 @@
+'use strict';
+
+const request = require('superagent');
+const server = require('../server.js');
+const serverToggle = require('../lib/server-toggle.js');
+const User = require('../model/user.js');
+const PhotoAlbum = require('../model/photoalbum.js');
+
+require('jest');
+
+const url = 'http://localhost:3000';
+
+const exampleUser = {
```

Comment:
Consider using faker to generate fake data rather than testing with a single set of data.